frontend/src/config/constants/formikConstants.tsx

```diff
@@ -0,0 +1,13 @@
+interface FormikInit {
```

Comment:
wht if i ll wrt cmmnts th sm wy? Rename to something more clear please

Reply:
Renamed